apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -33,6 +33,8 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
 
   require Utxo
 
+  @moduletag :exits
```

Comment:
leftover?

Note in general that `mix test test/exit_processor/core_test.exs` works, as does `mix test test/exit_processor/core_test.exs:123`, 123 being **any** line within test's body - for example line with the failing assertion puked out to the console by `ExUnit`

Reply:
done